Agents/FenlandTrajectoryAgent/README.md

```diff
@@ -0,0 +1,149 @@
+|------------|----------|-----------|-----------|--------|---------|--------|----------|
+| 2015/05/22 | 11:34:46 | 52.174374 | 0.137380  | 5.796  | 0       | 48.135 | 18.06    |
+| 2015/05/22 | 11:34:56 | 52.174315 | 0.137153  | 5.785  | 0       | 48.611 | 16.86    |
+| 2015/05/22 | 11:35:06 | 52.174194 | 0.137029  | 6.320  | 0       | 53.210 | 16.55    |
+| ...        | ...      | ...       | ...       | ...    | ...     | ...    | ...      |
+
+## 3.2 Building the Agent
+
+Simply execute the following command from a bash terminal in the same folder as this [README] to build the agent. This will build the Fenland-Trajectory-Agent local Docker Image. 
+
+```
+bash ./stack.sh build
+```
+
+In case of time out issues in automatically building the StackClients resource indicated in agent building logs, please try pulling the required stack-clients image first by execute `docker pull ghcr.io/cambridge-cares/stack-client:1.27.2` in the terminal before building the agent.
+
+## 3.3 Deploying the Agent
+### 1) Configuration and Adding Services
+
+Please place [FenlandTrajectoryAgent.json] in [stack manager configuration service directory]. Afterward, ensure that the 'fenland-trajectory-agent service' is included in your stack configuration files normally placed in [stack manager configuration directory]
+
+### 2) Starting with the Stack Manager
+To spin up the stack, both a `postgis_password` and `geoserver_password` file need to be created in the `Deploy/stacks/dynamic/stack-manager/inputs/secrets/` directory (see detailed guidance in the [Stack Manager README]).
+To spin up the stack, create `postgis_password` and `geoserver_password` files in the Deploy/stacks/dynamic/stack-manager/inputs/secrets/ directory. Detailed guidance can be found in the [Stack Manager README]. Thereafter，navigate to `Deploy/stacks/dynamic/stack-manager` and run the following command there from a *bash* terminal, Once the stack is up, the agent can be started within the stack autimatically, and will be available in the stack's drop-down container menu.
+
+```
+
+./stack.sh start <STACK NAME> <PORT>
+
+```
+
+# 4. Using the Agent
+
+The agent will automatically register a task upon startup to assimilate the data from the target GPS folder into the Knowledge Graph (KG). This background task can be triggered via an HTTP request after the agent has started, accessible at http://localhost:{Port}/fenland-trajectory-agent. Please replace {Port} with the numerical value of the port on which your stack is running.
+
+## Provided functionality
+
+The files for example http request are displayed at [SendHTTP] folder.
+
+<center>-------- Data Instantiation ------------</center>
+
+- Post request to load and preprocess all GPS trajectory files 
+> ./gpstasks/fenlandtrajectoryagent/load_and_preprocess
+
+- Post request to instantiate all GPS trajectory data to KG
+> ./gpstasks/fenlandtrajectoryagent/process_and_instantiate
```

Comment:
Please delete these two paths/routes here. They are no longer correct anyway. It is best to simply refer the reader to the http/curl files.

Reply:
These two routes were removed at commit [865d00cb10df9f472b2f2163b1878f1ac696be4c]